WrightTools/data/_colors.py

```diff
@@ -201,12 +201,12 @@ def from_COLORS(
         axes['m5'] = Axis(None, None, tolerance=15.0, file_idx=27, name='m5', label_seed=['5'])
         axes['m6'] = Axis(None, None, tolerance=15.0, file_idx=28, name='m6', label_seed=['6'])
         channels = collections.OrderedDict()
-        channels['ai0'] = Channel(None, 'V', file_idx=16, name='ai0', label_seed=['0'])
-        channels['ai1'] = Channel(None, 'V', file_idx=17, name='ai1', label_seed=['1'])
-        channels['ai2'] = Channel(None, 'V', file_idx=18, name='ai2', label_seed=['2'])
-        channels['ai3'] = Channel(None, 'V', file_idx=19, name='ai3', label_seed=['3'])
-        channels['ai4'] = Channel(None, 'V', file_idx=20, name='ai4', label_seed=['4'])
-        channels['mc'] = Channel(None, None, file_idx=21, name='array', label_seed=['a'])
+        channels['ai0'] = Channel(None, file_idx=16, name='ai0', label_seed=['0'])
+        channels['ai1'] = Channel(None, file_idx=17, name='ai1', label_seed=['1'])
+        channels['ai2'] = Channel(None, file_idx=18, name='ai2', label_seed=['2'])
+        channels['ai3'] = Channel(None, file_idx=19, name='ai3', label_seed=['3'])
+        channels['ai4'] = Channel(None, file_idx=20, name='ai4', label_seed=['4'])
+        channels['mc'] = Channel(None, file_idx=21, name='array', label_seed=['a'])
```

Comment:
Shall we move name to be a positional argument, now that it is required?

Reply:
I'm too lazy---these lines of code are not long for this world anyway